src/globals.ts

```diff
@@ -88,7 +88,11 @@ export async function createConnections(options?: ConnectionOptions[]): Promise<
     if (!options)
         options = await new ConnectionOptionsReader().all();
     const connections = options.map(options => getConnectionManager().create(options));
-    return Promise.all(connections.map(connection => connection.connect()));
+    // Do not use Promise.all or test 8522 will produce a dangling sqlite connection
+    for (const connection of connections) {
```

Comment:
Alternatively we can use Promise.allSettled(). But then only node version 12.9.0+ would work.